mmv1/third_party/terraform/services/sql/resource_sql_database_instance.go.tmpl

```diff
@@ -1713,6 +1758,42 @@ func expandPasswordValidationPolicy(configured []interface{}) *sqladmin.Password
 	}
 }
 
+{{- if ne $.TargetVersionName "ga" }}
+func expandConnectionPoolConfig(configured []interface{}) *sqladmin.ConnectionPoolConfig {
+	if len(configured) == 0 || configured[0] == nil {
+		return nil
+	}
+
+	_connectionPoolConfig := configured[0].(map[string]interface{})
+	return &sqladmin.ConnectionPoolConfig{
+		ConnectionPoolingEnabled: _connectionPoolConfig["connection_pooling_enabled"].(bool),
+		Flags:                    expandConnectionPoolFlags(_connectionPoolConfig["flags"]),
+		ForceSendFields:          []string{"ConnectionPoolingEnabled"},
```

Comment:
Is `ConnectionPoolingEnabled` required in the API request? Is it allowed to not send this field on API side when it is false or not set?

Reply:
I confirmed API accepts a request without connection_pooling_enabled field.